scale_build/bootstrap/cache.py

```diff
@@ -52,6 +54,21 @@ def mirror_cache_intact(self):
             self.logger.debug('Upstream repo changed! Removing squashfs cache to re-create.')
             intact = False
 
+        if intact:
+            self.restore_cache(self.chroot_basedir)
+            if os.path.exists(
```

Comment:
I don’t like that checked file names are hardcoded here. Can we please move them away to the same module that compares them?

Also what if some files exist and others don’t? Shouldn’t that be an impossible situation (that causes an exception)?